### PR TITLE
zh-cn: update the translation of "HTTP Overview"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of "HTTP Overview"

### Related issues and pull requests

The images have been replaced in en-US: mdn/content#34251

### CODE REVIEW

**Comment by jasonren0403** [comment]
seems that images from shared assets don't load in preview server 🤔 

image

File: files/zh-cn/web/http/overview/index.md [hunk 1]
```diff
@@ -31,19 +33,19 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
 
 浏览器**总是**首先发起请求的那个实体，永远不会是服务端（不过，后来已经加入了一些机制，能够模拟出由服务端发起的消息）。
 
-为了展现一个网页，浏览器需要发送最初的请求来获取描述这个页面的 HTML 文档。接着，解析文档，并发送数个其他请求，响应地获取可执行脚本、展示用的布局信息（CSS）以及其他页面内的资源（一般是图片和视频等）。然后，浏览器将这些资源整合到一起，展现出一个完整的文档，即 Web 页面。之后的阶段，浏览器中执行的脚本可以获取更多资源，同时浏览器相应地更新网页。
+为了展现一个网页，浏览器需要发送最初的请求来获取描述这个页面的 HTML 文档。接着，解析文档，并发送数个其他请求，响应地获取可执行脚本、展示用的布局信息（CSS）以及其他页面内的资源（一般是图片和视频等）。然后，Web 浏览器将这些资源整合到一起，展现出一个完整的文档，即网页。之后的阶段，浏览器中执行的脚本可以获取更多资源，并且浏览器会相应地更新网页。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:36]:
```suggestion
为了展现一个网页，浏览器需要发送最初的请求来获取描述这个页面的 HTML 文档。接着，解析文档，并发送数个其他请求，相应地获取可执行脚本、展示用的布局信息（CSS）以及其他页面内的子资源（一般是图片和视频等）。然后，Web 浏览器将这些资源整合到一起，展现出一个完整的文档，即网页。在之后的阶段，浏览器中执行的脚本可以获取更多资源，并且浏览器会相应地更新网页。
```

File: files/zh-cn/web/http/overview/index.md [hunk 2]
```diff
@@ -31,19 +33,19 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
...
 
-一个 Web 页面是一个超文本文档。这意味着有一部分展示的内容会是链接——可以被激活（通常是点击鼠标）来获取一个新的网页——用户可以通过这些链接指示用户代理并进行网络浏览。浏览器会将收到的指示转换成 HTTP 请求，并进一步解析 HTTP 响应，向用户提供清楚的回复。
+网页是超文本文档。这意味着有一部分展示的内容会是链接——可以通过激活（通常是点击鼠标）来获取一个新的网页——用户可以通过这些链接指示用户代理并进行 Web 浏览。浏览器会将收到的指示转换成 HTTP 请求，并进一步解析 HTTP 响应，向用户提供清楚的响应。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:38]:
```suggestion
网页是超文本文档。这意味着有一部分展示的内容会是链接——可以通过激活（通常是点击鼠标）来获取一个新的网页——用户可以通过这些链接指示用户代理并进行 Web 浏览。浏览器会将收到的指示转换成 HTTP 请求，并进一步解析 HTTP 响应，向用户提供清晰的响应。
```

File: files/zh-cn/web/http/overview/index.md [hunk 3]
```diff
@@ -31,19 +33,19 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
...
 
-### Web 服务端
+### Web 服务器
 
-在上述通信过程的另一侧是服务端，它*负责提供*客户端所请求的文档。一个服务端可以假装表现为仅有一台机器，但实际上，它可以是共享负载的一组服务器集群（负载均衡）或是其他类型的软件（如缓存、数据库服务、电商服务等），按需完整或部分地生成文档。
+在上述通信过程的另一侧是服务器，它*负责提供*客户端所请求的文档。一个服务器可以假装表现为仅有一台机器，但实际上，它可以是共享负载的一组服务器集群（负载均衡）或是其他类型的软件（如缓存、数据库服务、电商服务等），按需完整或部分地生成文档。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:42]:
```suggestion
在上述通信过程的另一侧是服务器，它负责*提供*客户端所请求的文档。服务器可以表现为仅有一台机器，但实际上，它可以是共享负载的一组服务器集群（负载均衡）或是其他类型的软件（如缓存、数据库服务、电商服务等），按需完整或部分地生成文档。
```

File: files/zh-cn/web/http/overview/index.md [hunk 4]
```diff
@@ -31,19 +33,19 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
...
 
 一个服务器可以不仅仅只有一台机器，而多个服务端软件实例也可部署在同一台机器上。利用 HTTP/1.1 和 {{HTTPHeader("Host")}} 标头，它们甚至可以共用同一个 IP 地址。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:44]:
```suggestion
服务器不一定只有一台机器，也可以在同一台机器上托管多个服务器软件实例。利用 HTTP/1.1 和 {{HTTPHeader("Host")}} 标头，它们甚至可以共用同一个 IP 地址。
```

File: files/zh-cn/web/http/overview/index.md [hunk 5]
```diff
@@ -59,41 +61,41 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
 
 ### HTTP 是可扩展的
 
-在 HTTP/1.0 中引入的 [HTTP 标头（header）](/zh-CN/docs/Web/HTTP/Headers)让协议扩展变得非常容易。只要服务端客户端之间对新标头的语义经过简单协商，新功能就可以被加入进来。
+在 HTTP/1.0 中引入的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)让协议扩展变得非常容易。只要服务端客户端之间对新标头的语义经过简单协商，新功能就可以被加入进来。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:64]:
```suggestion
在 HTTP/1.0 中引入的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)让该协议易于扩展和实验。只要服务器客户端之间对新标头的语义经过简单协商，新功能就可以被加入进来。
```

File: files/zh-cn/web/http/overview/index.md [hunk 6]
```diff
@@ -59,41 +61,41 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
...
+多年以来，良好的扩展性使得 HTTP 涉及到更多的 Web 功能与控制权。在 HTTP 诞生的早期，缓存和认证就可以由这个协议来处理了。而直到 2010 年，放行*同源限制*的能力才加入到协议中。
 
 以下是可以被 HTTP 控制的常见特性。
 
 - _[缓存](/zh-CN/docs/Web/HTTP/Caching)_：文档如何被缓存可以通过 HTTP 来控制。服务端能指示代理和客户端缓存哪些内容以及缓存多长时间，客户端能够指示中间的缓存代理来忽略已存储的文档。
-- _开放同源限制_：为了阻止网络窥听和其它侵犯隐私的问题，浏览器强制在不同网站之间做了严格分割。只有来自于**相同来源**（same origin）的网页才能够获取一个网页的全部信息。这种限制有时对服务器是一种负担，服务器的 HTTP 标头可以减弱此类严格分离，使得一个网页可以是由源自不同地址的信息拼接而成。某些情况下，放开这些限制还有安全相关的考虑。
+- _开放同源限制_：为了阻止网络窥听和其它侵犯隐私的问题，Web 浏览器强制在不同网站之间做了严格分割。只有来自于**相同来源**（same origin）的网页才能够获取一个网页的全部信息。这种限制有时对服务器是一种负担，服务器的 HTTP 标头可以减弱此类严格分离，使得一个网页可以是由源自不同地址的信息拼接而成。某些情况下，放开这些限制还有安全相关的考虑。
 - _认证_：一些页面可能会被保护起来，仅让特定的用户进行访问。基本的认证功能可以直接由 HTTP 提供，既可以使用 {{HTTPHeader("WWW-Authenticate")}} 或其他类似的标头，也可以用 [HTTP cookie](/zh-CN/docs/Web/HTTP/Cookies) 来设置一个特定的会话。
 - _[代理服务器和隧道](/zh-CN/docs/Web/HTTP/Proxy_servers_and_tunneling)_：
   服务器或客户端常常是处于内网的，对其他计算机隐藏真实 IP 地址。因此 HTTP 请求就要通过代理服务器越过这个网络屏障。并非所有的代理都是 HTTP 代理，例如，SOCKS 协议就运作在更底层。其他的协议，比如 ftp，也能够被这些代理处理。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:90]:
```suggestion
- _[代理服务器和隧道](/zh-CN/docs/Web/HTTP/Proxy_servers_and_tunneling)_：服务器或客户端常常是处于内网的，对其他计算机隐藏真实 IP 地址。因此 HTTP 请求就要通过代理服务器越过这个网络屏障。并非所有的代理都是 HTTP 代理，例如，SOCKS 协议就运作在更底层。其他的协议，比如 ftp，也能够被这些代理处理。
```

File: files/zh-cn/web/http/overview/index.md [hunk 7]
```diff
@@ -59,41 +61,41 @@ HTTP 是一个客户端—服务器协议：请求由一个实体，即用户代
...
 
 ## HTTP 流
 
-当客户端想要和服务端——不管是最终的服务端还是中间的代理——进行信息交互时，过程表现为下面几步：
+当客户端想要和服务器——不管是最终的服务器还是中间的代理——进行信息交互时，过程表现为下面几步：
 
-1. 打开一个 TCP 连接：TCP 连接被用来发送一条或多条请求，以及接受响应消息。客户端可能打开一条新的连接，或重用一个已经存在的连接，或者也可能开几个新的 TCP 连接连向服务端。
-2. 发送一个 HTTP 报文：HTTP 报文（在 HTTP/2 之前）是语义可读的。在 HTTP/2 中，这些简单的消息被封装在了帧中，这使得报文不能被直接读取，但是原理仍是相同的。
+1. 打开一个 TCP 连接：TCP 连接被用来发送一条或多条请求，以及接受响应消息。客户端可能打开一条新的连接，或重用一个已经存在的连接，或者也可能开几个新的与服务器的 TCP 连接。
+2. 发送一个 HTTP 报文：HTTP 报文（在 HTTP/2 之前）是人类可读的。在 HTTP/2 中，这些简单的消息被封装在了帧中，这使得报文不能被直接读取，但是原理仍是相同的。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:98]: 127行：人类可读

```suggestion
2. 发送一个 HTTP 报文：HTTP 报文（在 HTTP/2 之前）是人类可读的。在 HTTP/2 中，这些简单的消息被封装在了帧中，这使得报文不能被直接读取，但是原理仍是相同的。例如：
```

File: files/zh-cn/web/http/overview/index.md [hunk 8]
```diff
@@ -130,38 +132,38 @@ HTTP/1.1 以及更早的 HTTP 协议报文都是语义可读的。在 HTTP/2 中
...
 HTTP 请求的一个例子：
 
-![一个基础的 HTTP 请求](http_request.png)
+![带标头的 HTTP GET 请求概览](https://mdn.github.io/shared-assets/images/diagrams/http/overview/http-request.svg)
 
 请求由以下元素组成：
 
-- 一个 HTTP [方法](/zh-CN/docs/Web/HTTP/Methods)，通常是由一个动词，像 {{HTTPMethod("GET")}}、{{HTTPMethod("POST")}} 等，或者一个名词，像 {{HTTPMethod("OPTIONS")}}、{{HTTPMethod("HEAD")}} 等，来定义客户端执行的动作。典型场景有：客户端意图抓取某个资源（使用`GET`）；发送 [HTML 表单](/zh-CN/docs/Learn/Forms)的参数值（使用`POST`）；以及其他情况下需要的那些其他操作。
-- 要获取的那个资源的路径——去除了当前上下文中显而易见的信息之后的 URL，比如说，它不包括{{glossary("protocol","协议")}}（`http://`）、{{glossary("domain","域名")}}（这里的域名是`developer.mozilla.org`），或是 TCP 的{{glossary("port","端口")}}（这里的端口是 `80`）。
+- HTTP [方法](/zh-CN/docs/Web/HTTP/Methods)，通常是由一个动词，像 {{HTTPMethod("GET")}}、{{HTTPMethod("POST")}} 等，或者一个名词，像 {{HTTPMethod("OPTIONS")}}、{{HTTPMethod("HEAD")}} 等，来定义客户端执行的动作。典型场景有：客户端意图抓取某个资源（使用 `GET`）；发送 [HTML 表单](/zh-CN/docs/Learn/Forms)的参数值（使用 `POST`）；以及其他情况下需要的那些其他操作。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:139]:
```suggestion
- HTTP [方法](/zh-CN/docs/Web/HTTP/Methods)，通常是由一个动词，像 {{HTTPMethod("GET")}}、{{HTTPMethod("POST")}} 等，或者一个名词，像 {{HTTPMethod("OPTIONS")}}、{{HTTPMethod("HEAD")}} 等，来定义客户端执行的动作。典型场景有：客户端意图获取某个资源（使用 `GET`）；发送 [HTML 表单](/zh-CN/docs/Learn/Forms)的参数值（使用 `POST`）；以及其他情况下需要的那些其他操作。
```

File: files/zh-cn/web/http/overview/index.md [hunk 9]
```diff
@@ -130,38 +132,38 @@ HTTP/1.1 以及更早的 HTTP 协议报文都是语义可读的。在 HTTP/2 中
...
 
-![HTTP 响应的图片](http_response.png)
+![对 GET 请求的“200 OK”HTTP 响应（包括响应标头）的概览。](https://mdn.github.io/shared-assets/images/diagrams/http/overview/http-response.svg)
 
 响应报文包含了下面的元素：
 
 - HTTP 协议版本号。
-- 一个状态码（[status code](/zh-CN/docs/Web/HTTP/Status)），来指明对应请求已成功执行或是没有，以及相应的原因。
-- 一个状态信息，这个信息是一个不权威、简短的状态码描述
+- [状态码](/zh-CN/docs/Web/HTTP/Status)，来指明对应请求已成功执行或是没有，以及相应的原因。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:154]:
```suggestion
- [状态码](/zh-CN/docs/Web/HTTP/Status)，来指明对应请求已成功执行与否，以及不成功时相应的原因。
```

File: files/zh-cn/web/http/overview/index.md [hunk 10]
```diff
@@ -130,38 +132,38 @@ HTTP/1.1 以及更早的 HTTP 协议报文都是语义可读的。在 HTTP/2 中
...
+- 状态信息，这个信息是一个不权威、简短的状态码描述
```
Comment (jasonren0403) [inline files/zh-cn/web/http/overview/index.md:155]:
```suggestion
- 状态信息，这个信息是一个不权威、简短的状态码描述。
```

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

**Comment by yin1999** [comment]
> seems that images from shared assets don't load in preview server 🤔 
>
> image


Caused by csp policy. It might be fixed in future.

**Review by jasonren0403 — APPROVED** [review]
👍 Thank you @yin1999

**Pull request merged**